run.sh

```diff
@@ -2,9 +2,39 @@
 
 export HOME=/kibana
 
+: ${ELASTICSEARCH_URL:=http://localhost:9200}
 : ${ELASTICSEARCH_CERTIFICATE_VERIFICATION:=true}
+: ${ELASTICSEARCH_SSL_CA_CERT_PATH:=}
+: ${ELASTICSEARCH_USERNAME:=}
+: ${ELASTICSEARCH_PASSWORD:=}
+: ${XPACK_SECURITY_ENABLE:=false}
+: ${XPACK_SECURITY_ENCRYPTION_KEY:=$(uuidgen)}
```

Comment:
@vaijab One can always pass a fixed value in environment to ensure all instances use the same encryption key.